packages/zoe/test/unitTests/contracts/test-invitation-details.js

```diff
@@ -0,0 +1,76 @@
+import { test } from '@agoric/swingset-vat/tools/prepare-test-env-ava.js';
```

Comment:
_remark_

Thank you for creating this test, it helps me understand the problem very well.